mmseg/datasets/transforms/transforms.py

```diff
@@ -1296,7 +1296,7 @@ def transform(self, results: Dict) -> Dict:
                                            (self.edge_width, self.edge_width))
         edge = cv2.dilate(edge, kernel)
 
-        results['gt_edge'] = edge
+        results['gt_edge_map'] = edge
```

Comment:
Should also modify `gt_edge` to `gt_edge_map` in [docstring](https://github.com/open-mmlab/mmsegmentation/blob/dev-1.x/mmseg/datasets/transforms/transforms.py#L1248).

Reply:
done